.github/workflows/e2e-tests.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
 
   e2e-tests:
     name: Execute E2E Tests
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-22.04-16core
```

Comment:
Seems a bit high, you sure about this?

Reply:
per discussion over slack, changed it to 8 core and we will see in the future if its not enough, thanks for the discussion :)